src/amo/components/SecondaryHero/styles.scss

```diff
@@ -109,7 +107,6 @@
 .SecondaryHero-message-link,
 .SecondaryHero-module-description,
 .SecondaryHero-module-link {
-  overflow-wrap: anywhere;
   width: 100%;
```

Comment:
In this file, we might want to use `anywhere` but I am not sure. I'd say we only use one rule for now and see how it works.

Reply:
So all `overflow-wrap: anywhere;` rules back to where they were?

Reply:
That would be safer, yeah but if it works without, then there is not needed for that. I guess we can try without?